src/Museum/Museum.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace DinoApp\Museum;
+
+use DinoApp\Dinosaur\Dinosaur;
+
+class Museum
+{
+    /**
+     * Takes in an array of dinosaur objects and puts them into the format to be displayed on the homepage.
+     *
+     * @param array $dinos an array of Dinosaur objects.
+     * @return string The html to display the dinosaurs on the homepage.
+     */
+    public static function displayAllDinos(array $dinos) :string
+    {
+        $output='';
+        foreach ($dinos as $dino){
+            if($dino instanceof Dinosaur){
+                $output .= '<style> .dino'. $dino->getId().'{background-image: url(' . $dino->getImageUrl() .');} </style>';
```

Comment:
remove inline style tag as main has it as a style property.

Reply:
This was a point from Ash about having a style attribute instead of a tag so there are fewer elements in the DOM :)